package.json

```diff
@@ -10,9 +10,12 @@
   "dependencies": {
     "grommet": "^2.17.2",
     "grommet-icons": "^4.5.0",
-    "next": "10.2.0",
+    "next": "latest",
```

Comment:
I think we can leave the version set to a specific number and then the user can update to whichever version they want. I don't think we want to have this set to "latest" if it is being used in production.

Reply:
@jcfilben That's a good point. But this "latest" will only affect when the project has no `package-lock.json` neither `yarn.lock`, otherwise, the version downloaded when users run `npm install` or `yarn` will be the version set on the lock files.

This ensures when users generate the project from this boilerplate, they will have the latest Next.js version, but when they re-download the dependencies it will not affect the version.

What do you think? Should I change it?

Reply:
Ok makes sense!